test/integration/crud/bulk.test.js

```diff
@@ -22,6 +27,34 @@ describe('Bulk', function () {
   before(function () {
     return setupDatabase(this.configuration);
   });
+  describe('class BulkOperationBase', () => {
+    context('.raw()', () => {
+      it('should throw a MongoInvalidArgument error when called with an undefined operation', async function () {
+        const client = this.configuration.newClient();
```

Comment:
Last comment - I think we should initialize and close the client in a beforeEach hook and an afterEach hook respectively.  This test uses a shared variable and a before/after Each hook to initialize it and close it for each test as an example - https://github.com/mongodb/node-mongodb-native/blob/a880eccade5491a0687af3f5c8c14d427e844995/test/integration/server-selection/server_selection_operation_count.prose.test.ts#L89